mathics/doc/doc_entries.py

```diff
@@ -394,11 +391,16 @@ def compare_out(self, outs: tuple = tuple()) -> bool:
             # Mismatched number of output lines, and we don't have "..."
             return False
 
+        # Python 3.13 replaces tabs by a single space in docstrings.
```

Comment:
Python 3.13 converts tab characters in docstrings to simple spaces. This "hack" helps to keep the doctests compatible with this change.